### PR TITLE
add search option for profile

### PR DESCRIPTION
**Pull request by btry**
this itemtype was missing a search option for ID

| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | 
| Fixed tickets | #5245

### CODE REVIEW

**Comment by trasher** [comment]
RC2 has been released already, this is too late for 9.4.0. This should target 9.4.1 and be merged after 9.4.0 has been released :)

**Comment by trasher** [comment]
9.4.0 has been released. You hav to rebase and create a specific 9.4.& update file for changes ;)

**Comment by btry** [comment]
@trasher @cedric-anne 

I did not add -dev to the GLPI version. This prevents GLPI_PREVER to be set, causing #5362 

This PR is WIP until #5362 is closed

**Comment by trasher** [comment]
`-dev` should be used for major only, no need here

File: inc/update.class.php [hunk 1]
```diff
@@ -434,11 +434,17 @@ public function doUpdates($current_version = null) {
...
          case "9.3.2":
          case "9.3.3":
-         case GLPI_PREVER:
             include_once("{$updir}update_93_94.php");
             update93to94();
             break;
 
+         case "9.4":
+         case "9.4.0":
+         case GLPI_PREVER:
```
Comment (trasher) [inline inc/update.class.php]: See my proposal on related ticket to avoid not defined constant

**Comment by btry** [comment]
> `-dev` should be used for major only, no need here

Ok, then I may remove the WIP

File: inc/update.class.php [hunk 2]
```diff
@@ -434,11 +434,17 @@ public function doUpdates($current_version = null) {
 
          case "9.3.2":
          case "9.3.3":
-         case GLPI_PREVER:
             include_once("{$updir}update_93_94.php");
             update93to94();
             break;
 
+         case "9.4":
```
Comment (trasher) [inline inc/update.class.php]: There is no "9.4" release. I've rebased and did the change; but I'm not able to push on your branch...

File: inc/update.class.php (hunk 1 above)
Comment (trasher) [inline inc/update.class.php]: please just drop this one.

File: inc/update.class.php (hunk 2 above)
Comment (trasher) [reply inc/update.class.php]: see https://github.com/glpi-project/glpi/compare/9.4/bugfixes...trasher:fix_profile_search_option

**Comment by trasher** [comment]
Replaced with #5401

**Pull request closed without merging**